package.json

```diff
@@ -14,6 +14,7 @@
         "publish-lib": "npm publish ./dist",
         "start": "rollup -c -w",
         "test": "jest --coverage",
+        "test:codemods": "jest --coverage tests/codemod",
```

Comment:
this change can be reverted as we can already pass the file path to the jest command